addOns/pscanrules/src/main/javahelp/org/zaproxy/zap/extension/pscanrules/resources/help/contents/pscanrules.html

```diff
@@ -287,7 +287,13 @@ <H2>PII Disclosure</H2>
 At MEDIUM and HIGH threshold it attempts to use three characters of context on each side of potential matches to exclude matches within decimal like content.
 At LOW threshold, alerts will be raised for such matches.
 <p>
-PDFs are only evaluated at LOW threshold.
+At MEDIUM and HIGH Threshold, the following content types are evaluated:
+<ul>
+    <li>HTML</li>
+    <li>JSON</li>
+    <li>XML</li>
+</ul>
+All other content types are evaluated at LOW threshold.
```

Comment:
This assertion is not correct and contradicts earlier description (i.e. `images and CSS are ignored`).

Reply:
I'd suggest remove the earlier description, to avoid duplication.

Reply:
No, I mean ` (images and CSS are ignored)` from line 287.